src/js/views/patients/schedule/schedule_views.js

```diff
@@ -294,6 +294,10 @@ const ScheduleListView = CollectionView.extend({
       return !view.isEmpty();
     }
 
+    if (view.model.get('date') === 'null') {
+      return false;
+    }
+
```

Comment:
When an action has no due date, its `date` value is equal to the `"null"` string. This code just adds a check for that and tells the `ScheduleListView` to not include that action in the worklist if that's the case.

This change will apply to both the normal and reduced versions of the schedule.

Reply:
A ... string `'null'` ... ? That doesn't seem right.

Reply:
Yeah it's odd, but it's a result of the `groupBy` for a case that's not supposed to happen.  I think we need a comment here as otherwise I'll be tempted to remove the string in the future.  This is a key from the `groupBy`

Reply:
Also @shadowhand this was a quick way to solve an edge case, but attempting to leave the `max` null on the endpoint didn't seem to work for not getting actions with `due_dates`.  Since this situation is apparently not a use case, I wasn't too worried about a small FE filter solving the issue vs looking into the endpoint further, but probably figured it might be worth mentioning.